c-realV2.py

```diff
@@ -2328,7 +2328,8 @@ def wrapper(self, *args, **kwargs):
         exit()
     client.run(settings['token'])
 except discord.PrivilegedIntentsRequired:
-    print('PrivilegedIntentsRequired: This field is required to request for a list of members in the discord server that the bot is connected to. Watch https://youtu.be/DXnEFoHwL1A?t=44 to see how to turn on the required field.')
+    print(chr(27) + "[2J")
```

Comment:
This might not work on some terminals

e.g. legacy windows terminal (windows console host)

Maybe just use the good old `os.system("clear")` and `os.system("cls")`?